### DISCUSSION TITLE
add 'show converted' for stable verison

### DISCUSSION THREAD

**Discussion by mas-alone** [ppy/osu#19675] [Ideas] [answered]
![微信截图20220810135506](https://user-images.githubusercontent.com/52590027/183825948-1de5b291-ee76-4563-b42d-98eec79f61ec.png)
A lot of Std mode beatmap will be automatically converted into Mania mode. 
It's nice to have a separate switch in the lazer for players to decide if they want to display them.
Is it possible to implement this feature in stable? Because a lot of converted beatmap is not logical.

**Comment by smoogipoo** [comment, marked as answer]
For osu-stable issues: https://github.com/ppy/osu-stable-issues/

But osu-stable is feature locked, so no. You can use `mode=m`.

**Reply by mas-alone** [reply]
Thank you for reply ^ ^
